requirements.txt.py

```diff
@@ -0,0 +1,12 @@
+XXXXXXXXXXXXX
```

Comment:
What are these symbols?

Reply:
This file should not have exists and will be removed.